modules/ai/assets/js/editor/components/prompt-history/context/prompt-history-context.js

```diff
@@ -0,0 +1,133 @@
+import { createContext, useContext, useEffect, useState } from 'react';
+import { HISTORY_TYPES } from '../history-types';
+import { vh } from '../../../helpers';
+
+export const PromptHistoryContext = createContext( {} );
+
+const DIALOG_CONTAINER_SELECTOR = '.e-ai-dialog-content';
+
+const getContainer = () => {
+	return document.querySelector( DIALOG_CONTAINER_SELECTOR );
+};
+
+const getContainerHeight = () => {
+	const contentContainer = document.querySelector( DIALOG_CONTAINER_SELECTOR );
+	return contentContainer?.clientHeight;
+};
+
+const getContentHeight = () => {
+	const contentContainer = document.querySelector( DIALOG_CONTAINER_SELECTOR );
+	return contentContainer.children[ 0 ].clientHeight + 50;
+};
+
+const setContentHeight = ( height ) => {
+	const contentContainer = document.querySelector( DIALOG_CONTAINER_SELECTOR );
+
+	if ( 'auto' === height ) {
+		return contentContainer.style.height = 'auto';
+	}
+
+	contentContainer.style.height = `${ height }px`;
```

Comment:
1. did you mean to return this value?
contentContainer.style.height = `${ height }px`;

ternary will be better her 

`return contentContainer.style.height = 'auto' === height ? 'auto' : `${ height }px`;`

But its flavor.